service/controller/resource/workermigration/internal/tenantclient/spec.go

```diff
@@ -0,0 +1,12 @@
+package tenantclient
```

Comment:
you can now use the new package in `pkg/tenantcluster`

Reply:
Done.